sdk/test/opentelemetry/sdk/trace/tracer_test.rb

```diff
@@ -227,7 +227,7 @@
     end
 
     it 'creates a span with all supplied parameters' do
-      links = [OpenTelemetry::Trace::Link.new(context)]
+      links = [OpenTelemetry::Trace::Link.new(span_reference)]
```

Comment:
I think we need to change this too - argument should be `span_context` instead of `context`.